cluster-api/cloud/google/machinesetup/config_types.go

```diff
@@ -0,0 +1,118 @@
+package machinesetup
+
+import (
+	"fmt"
+	"github.com/ghodss/yaml"
+	"io"
+	"io/ioutil"
+	clustercommon "k8s.io/kube-deploy/cluster-api/pkg/apis/cluster/common"
+	clusterv1 "k8s.io/kube-deploy/cluster-api/pkg/apis/cluster/v1alpha1"
+	"k8s.io/kube-deploy/cluster-api/util"
+	"os"
```

Comment:
These imports should be grouped differently. See https://github.com/golang/go/wiki/CodeReviewComments#imports